HCSStarRatingView/HCSStarRatingView.m

```diff
@@ -267,6 +308,8 @@ - (void)drawRect:(CGRect)rect {
     CGFloat availableWidth = rect.size.width - (_spacing * (_maximumValue - 1)) - 2;
     CGFloat cellWidth = (availableWidth / _maximumValue);
     CGFloat starSide = (cellWidth <= rect.size.height) ? cellWidth : rect.size.height;
+    starSide = (self.shouldUseImages)? starSide : (starSide - _starBorderWidth);
```

Comment:
Missing space before `?`

Also, why is here exactly? (seriously, I'm asking out of ignorance 😆 and only proves how much I need to take another good look at my own code again)